src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private LocalDateTime orderDate;
+    @OneToMany
+    private List<Ticket> tickets;
+    @ManyToOne
```

Comment:
By default, @ManyToOne relationships use FetchType.EAGER, which means that the associated entity is loaded immediately along with the owning entity. However, it is preferable to use FetchType.LAZY to improve performance by loading the associated entity only when it is accessed. 
You can find more info here: https://thorben-janssen.com/common-hibernate-mistakes-cripple-performance/#Mistake_1_Use_Eager_Fetching